core/embed/sys/mpu/stm32u5/mpu.c

```diff
@@ -137,16 +137,10 @@ _Static_assert(NORCOW_SECTOR_SIZE == STORAGE_2_MAXSIZE, "norcow misconfigured");
 
 #define OTP_AND_ID_SIZE 0x800
 
-// clang-format on
-extern uint8_t boot_args_start;
-#define BOOTARGS_START ((uint32_t) & boot_args_start)
-
 #ifdef KERNEL
 
 extern uint8_t _uflash_start;
 extern uint8_t _uflash_end;
-#define KERNEL_RAM_U_START (KERNEL_RAM_START + KERNEL_RAM_SIZE)
-#define KERNEL_RAM_U_SIZE KERNEL_U_RAM_SIZE
 #define KERNEL_FLASH_U_START (uint32_t) & _uflash_start
```

Comment:
We can remove `KERNEL_FLASH_U_START`/`SIZE` as well, seems it's no longer used.

Reply:
no these are still used in KERNEL_FLASH_START and COREAPP_FLASH_START calculations below - since we decided to include the U_FLASH into the coreapp mpu region along with the empty space between kernel and coreapp to fix the fw hashing.